_posts/2021-04-13-upgrade-notification-for-ase.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Upgrade Notifications for App Service"
+author_name: "Michimune Kohno and Sanghmitra Gite"
+toc: true
+toc_sticky: true
+tags:
+    - Upgrade notifications
+    - Azure Monitor
+    - App Service Environment
+    - Logic Apps
+---
+
+We are happy to announce that the upgrade notifications for App Service Environment is available. With an upgrade notification you will be able to receive email, SMS text, or take action before a platform upgrade actually starts.
```

Comment:
'be able to receive email or SMS to allow you to take action before a platform upgrade starts if necessary.'

Reply:
Or actually, to make sure I understand correctly, can a user define a custom action? I could've misunderstood this.

Reply:
Ah ok. I meant automation like Azure Function by "take action", but your comment is valid. I will update the text to include your point.

Reply:
Updated